### PR TITLE
github-cli: install shell completion

### PR DESCRIPTION
**Pull request by gbrlsnchs**

### CODE REVIEW

**Comment by gbrlsnchs** [comment]
CI is throwing the error `cannot execute binary file: Exec format error` for ARM architectures. Does this mean this package was silently passing but was compiling a non-functional non-ARM binary?

**Comment by ericonr** [comment]
No. The ARM binaries are cross compiled, so the host machine _can't_ execute them. To generate completions, use `vtargetrun`, as in the `rustup` template.

**Comment by gbrlsnchs** [comment]
> No. The ARM binaries are cross compiled, so the host machine can't execute them. To generate completions, use vtargetrun, as in the rustup template.

Duh, my bad. And TIL about `vtargetrun`, thanks!

It now seems to be working but some CI jobs have timed out. :disappointed:

**Comment by gbrlsnchs** [comment]
@ericonr, do you have permission to rerun the CI jobs? I think everything's fine and working, it has just timed out.

**Comment by ericonr** [comment]
@gbrlsnchs you can rebase your branch and force push it. That should restart CI.

**Comment by gbrlsnchs** [comment]
All right, @ericonr, all good now, thanks for the tip.

File: srcpkgs/github-cli/template [hunk 1]
```diff
@@ -22,6 +23,19 @@ pre_build() {
 	fi
 }
 
+post_build() {
+	gopath=$(go env GOPATH)
+	gh=$(find $gopath/bin -name gh)
```
Comment (ericonr) [inline srcpkgs/github-cli/template:28]: We set `GOPATH` for the build, so it should work just as well with `find $GOPATH/bin`

**Comment by gbrlsnchs** [comment]
@ericonr All good now.

**Pull request merged**